aospy_synthetic/region.py

```diff
@@ -0,0 +1,103 @@
+"""region.py: Region class and region_inst()."""
```

Comment:
So nothing's been changed in `Region`, right?  Just including it for first time here?